app/src/components/widgets/multiLevelWidgets/componentHealthCheck/componentHealthCheck.jsx

```diff
@@ -0,0 +1,272 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import isEqual from 'fast-deep-equal';
+import classNames from 'classnames/bind';
+import {
+  COLOR_BURGUNDY,
+  COLOR_CHERRY,
+  COLOR_DEEP_RED,
+  COLOR_DULL_RED,
+  COLOR_PASSED,
+  COLOR_DULL_GREEN,
+} from 'common/constants/colors';
+import { ScrollWrapper } from 'components/main/scrollWrapper';
+import { NoDataAvailable } from 'components/widgets/noDataAvailable';
+import { ComponentHealthCheckLegend } from './legend/componentHealthCheckLegend';
+import { GroupsSection } from './groupsSection/groupsSection';
+import styles from './componentHealthCheck.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  failedGroupsTitle: {
+    id: 'ComponentHealthCheck.failedGroupsTitle',
+    defaultMessage: 'Failed',
+  },
+  passedGroupsTitle: {
+    id: 'ComponentHealthCheck.passedGroupsTitle',
+    defaultMessage: 'Passed',
+  },
+});
+
+const MAX_PASSING_RATE_VALUE = 100;
```

Comment:
Duplicates [here](https://github.com/reportportal/service-ui/pull/2009/files#diff-4207fb0fd9afa0ad22a15718de40e919R20).
Please move it to the common widget constant.